pkg/virt-controller/services/multus_annotations.go

```diff
@@ -75,6 +88,22 @@ func GenerateMultusCNIAnnotationFromNameScheme(namespace string, interfaces []v1
 	return "", nil
 }
 
+func newBindingPluginMultusAnnotationData(kvConfig *v1.KubeVirtConfiguration, pluginName, namespace, networkName string) (*networkv1.NetworkSelectionElement, error) {
+	plugin := netbinding.ReadNetBindingPluginConfiguration(kvConfig, pluginName)
+	if plugin == nil {
+		return nil, fmt.Errorf("couldnt find network binding plugin '%s' in Kubevirt configuration", pluginName)
+	}
+
+	netAttachDefNamespace, netAttachDefName := getNamespaceAndNetworkName(namespace, plugin.NetworkAttachmentDefinition)
+	return &networkv1.NetworkSelectionElement{
+		Namespace: netAttachDefNamespace,
+		Name:      netAttachDefName,
+		CNIArgs: &map[string]interface{}{
+			"vm-network-name": networkName,
```

Comment:
This will become a formal API between kubevirt and its binding CNI(s).
I think it needs to be a constant with a good description. In the future, it should probably be elevated to a binding API package.

Reply:
Done